pypdf/filters.py

```diff
@@ -365,131 +364,6 @@ def decode(
         return b"".join(lst)
 
 
-class LZWDecode:
```

Comment:
AFAIK our deprecation policy requires to keep this compatibility layer until we release version 7.0.

Reply:
I've added it back, but the implementation now is the new _codecs.py one :-)

Reply:
Thanks. I have just removed some outdated comments. I guess we are going to keep the old class for now without any deprecation process?